core/src/main/java/org/mule/runtime/core/DefaultEventContext.java

```diff
@@ -161,4 +174,91 @@ public void subscribe(Subscriber<? super Event> s) {
     subscribers.add(s);
     monoProcessor.subscribe(s);
   }
+
+  private static class ChildEventContext implements EventContext, Serializable {
+
+    private static final long serialVersionUID = 1054412872901205234L;
+
+    private transient MonoProcessor<Event> monoProcessor = MonoProcessor.create();
+    private List<Subscriber> subscribers = new ArrayList<>();
```

Comment:
No because that would rule out the use of reactor/rxjava as well as be quite limiting.  Also regardless of reactor/rxJava a HTTP callback typically wouldn't be serializable either as it needs to maintain reference to connection.